pwiz_tools/Skyline/EditUI/AssociateProteinsDlg.cs

```diff
@@ -275,7 +278,10 @@ private void rbCheckedChanged(object sender, EventArgs e)
         private void tbxFastaTargets_TextChanged(object sender, EventArgs e)
         {
             if (File.Exists(tbxFastaTargets.Text))
+            {
+                Settings.Default.LastProteinAssociationFastaFilepath = tbxFastaTargets.Text;

```

Comment:
Normally, we would save a value in OkDialog() after we are sure everything is valid, to avoid saving a value the user didn't really end up using, e.g. I typed "foobar.fasta" realized I was wrong and canceled out of the form.

Reply:
OK, makes sense.